### PR TITLE
[Resources and support] Fix for search bar submit endpoints

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
These URLs need trailing slashes during submit, otherwise they cause a redirect and lose the query params.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/15825#event-3962801562
## Testing done
Tested the submit locally and on domain using dev tools

## Screenshots
N/A

## Acceptance criteria
- [ ] Search bar carries the query over

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function SearchBar({ onSearch, userInput, onInputChange }) {
         onSubmit={isGlobalSearch ? null : onSubmit}
         id="resources-support-search"
         method="get"
-        action={isGlobalSearch ? '/search' : '/resources/search'}
+        action={isGlobalSearch ? '/search/' : '/resources/search/'}
```
Comment (kelsonic) [inline src/applications/resources-and-support/components/SearchBar.jsx:21]: You could use my constant here for `/resources/search/` if you want! 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/14865/files#diff-ef6a912a8c059f7814bf5b6bd82acd8ea06d150a6235a28b4d59289e890d640eR1

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/resources-and-support/components/SearchBar.jsx (hunk 1 above)
Comment (kelsonic) [reply src/applications/resources-and-support/components/SearchBar.jsx:21]: Might be nice to put `/search/` in that constants file too, maybe calling it something like `VAGOV_SEARCH_PATHNAME`? That's a terrible name but you get the point 😅

Comment (ncksllvn) [reply src/applications/resources-and-support/components/SearchBar.jsx:21]: Oh yeah I suppose we could import the `manifest.json` from the different apps...good idea. I'll update

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 2]
```diff
@@ -18,7 +21,11 @@ export default function SearchBar({ onSearch, userInput, onInputChange }) {
         onSubmit={isGlobalSearch ? null : onSubmit}
         id="resources-support-search"
         method="get"
-        action={isGlobalSearch ? '/search' : '/resources/search'}
+        action={
+          isGlobalSearch
+            ? `${searchSettings.rootUrl}/`
+            : `${resourcesSettings.rootUrl}/`
+        }
```
Comment (ncksllvn) [inline src/applications/resources-and-support/components/SearchBar.jsx:28]: @kelsonic made that change fysa - look good?

Comment (kelsonic) [reply src/applications/resources-and-support/components/SearchBar.jsx:28]: Oh wow! Nice! I'm going to add this to my PR too, thanks!

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import React, { useState } from 'react';
 
+import resourcesSettings from '../manifest.json';
+import searchSettings from 'applications/search/manifest.json';
```
Comment (U-DON) [inline src/applications/resources-and-support/components/SearchBar.jsx:4]: :+1:

**Review by U-DON — APPROVED** [review]

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

**Pull request merged**